docs/checkout.md

```diff
@@ -20,12 +20,12 @@ sylius_gtm_enhanced_ecommerce:
                         selector: "form[name=sylius_checkout_address]"
                 3:
                     -
+                        option: "enhancedEcommerceCheckoutGetChoiceVal"
```

Comment:
Not sure why, but I prefer `enhancedEcommerceCheckoutGetChoiceValue`. All words are written, only the last one is a abbrev.. :)